ncm-spma/src/main/pan/components/spma/config.pan

```diff
@@ -5,10 +5,10 @@
 
 unique template components/${project.artifactId}/config;
 
-variable CONFIG_MODULES_CONFIG_SUFFIX ?= 'rpm';
+variable SPMA_BACKEND ?= 'yum';
```

Comment:
Why do we have the SPMA_BACKEND variable as well as the /software/components/spma/packager element? Can we just use the latter instead of the global?